src/main/resources/css/MainWindow.css

```diff
@@ -1,10 +1,10 @@
 .background {
-    -fx-background-color: #2b5c7b;
-    background-color: #2b5c7b; /* Used in the default.html file */
+    -fx-background-color: #aec6cf;
```

Comment:
I would like to check if we are changing the colour scheme of the UI. I think `#2b5c7b` is the one we are using in our mock-up.